git-webui/release/share/git-webui/webui/js/git-webui.js

```diff
@@ -1256,11 +1256,11 @@ webui.StashCommitView = function(stashView) {
     var self = this;
 
     self.update = function(entry) {
-        if (currentCommit == entry.commit) {
-            // We already display the right data. No need to update.
-            return;
+        // if (currentCommit == entry.commit) {
```

Comment:
Remove dead code, don't just comment it out!

Reply:
Still relevant

Reply:
This is in the release file. I just didn't recompile after removing the dead code. Is that fine?